backend/controllers/search.js

```diff
@@ -37,22 +40,22 @@ const externalSearchFactory = (
   }
 
   const results = async (req, res) => {
-    const { user, groupCode, roleCode, size, page } = req.query
+    const { user, groupCode, roleCode, size, page, offset } = req.query
```

Comment:
Prison API uses `offset` instead, ideally we would convert the Prison API stuff to JPA and then can use `page`, but little steps